modules/gapi/src/backends/ie/giebackend.cpp

```diff
@@ -239,14 +239,6 @@ struct IEUnit {
             // Still, constant data is to set only once.
             this_request.SetBlob(p.first, wrapIE(p.second.first, p.second.second));
         }
-        // Bind const data to infer request
-        for (auto &&p : params.const_inputs) {
-            // FIXME: SetBlob is known to be inefficient,
-            // it is worth to make a customizable "initializer" and pass the
-            // cv::Mat-wrapped blob there to support IE's optimal "GetBlob idiom"
-            // Still, constant data is to set only once.
-            this_request.SetBlob(p.first, wrapIE(p.second.first, p.second.second));
-        }
```

Comment:
Wy it is removed from ie backend? Did you validate the existing samples requiring it?

Reply:
This is duplicate code. It may have already been fixed.

Reply:
No, it isn't fixed. 235 line - same code.

Reply:
Oh I see now. Hmm strange probably some merge artifact. Got it, thanks!

Reply:
There is no such duplication on the master branch
Please cleanup patch (squash and rebase).